vm/src/stdlib/os.rs

```diff
@@ -1575,6 +1575,11 @@ mod posix {
                     PyPathLike::try_from_object(&vm, k)?,
                     PyPathLike::try_from_object(&vm, v)?,
                 );
+
+                if key.path.display().to_string().contains('=') {
```

Comment:
is `display()` required here? I think any `Display` is `ToString`.
https://doc.rust-lang.org/std/string/trait.ToString.html#impl-ToString-3

Reply:
Isn't `display()` required to use `contains()`, which is implemented in `String`? AFAIK, Direct conversion from `PathBuf` to `String` creates more room for exceptions, due to how PathBuf and String handles string literals differently.

Reply:
Ah, you are right. I thought display is a trait method but it is a impl method https://doc.rust-lang.org/std/path/struct.Path.html#method.display

By checking a bit more, I now think Path -> OsStr -> CStr conversion is correct for this case. display() make it convert to `str` which implies utf-8, but Path and CStr actually doesn't need to be utf-8 guaranteed.
Because this is not a part of this PR but glitch from other changes before, It is ok for now as it is.